publisher/product/ontology/build-about.sh

```diff
@@ -48,11 +40,11 @@ __HERE__
     "${JENA_ARQ}" \
       --data="${tmpAboutFileProd}" \
       --query="${SCRIPT_DIR}/lib/echo.sparql" \
-      --results=RDF > "${tag_root}/LoadFIBOProd.rdf"
+      --results=RDF > "${tag_root}/Load${ONTPUB_FAMILY}Prod.rdf"
     # 2> "${TMPDIR}/err.tmp"
 
     if [ -s "${TMPDIR}/err.tmp" ] ; then
-      warning "no RDF XML output generated.  Use LoadFIBOProd.ttl file instead"
+      warning "no RDF XML output generated.  Use Load${ONTPUB_FAMILY}Prod.ttl file instead"
```

Comment:
Should be uppercase:
`warning "no RDF XML output generated. Use Load${ONTPUB_FAMILY^^}Prod.ttl file instead"`